### DISCUSSION TITLE
Could not parse base64Decode: offset must be positive, non-zero values only

### DISCUSSION THREAD

**Discussion by muskan399** [google/gonids#191] [General]
While parsing a rule with `..   offset 0; ..`, getting could not parse base64Decode: offset must be positive, non-zero values only as error

**Comment by duanehoward** [comment]
This is https://github.com/google/gonids/issues/175 feel free to follow up there. Pull requests are welcome. The main piece that's missing is to confirm either with the Suricata dev team or by validating the source code that this is in fact allowed. (I suspect it is)
